Application.mk

```diff
@@ -262,3 +262,7 @@ distclean:: clean
 	$(call DELFILE, .depend)
 
 -include Make.dep
+
+ifeq ($(CONFIG_INTERPRETERS_WAMR)_$(WASM_BUILD),y_y)
```

Comment:
i'd suggest:

* separate wamr-specific stuff (wamrc) and wasm-generic stuff (wasi-sdk)
* control only the former with CONFIG_INTERPRETERS_WAMR

Reply:
Done